### ISSUE TITLE
Interactive CLI

### ISSUE THREAD

**Issue by ok2ju** [jszombies/create-zombie-lib#4]
Maybe we should make this CLI interactive? I mean console will ask you questions about template type, project name, etc...

**Comment by alehatsman** [comment]
Yes for sure. I just wanted to make a minimal working solution as fast as possible. We can add such features now.
